content/docs/esc/faq.md

```diff
@@ -13,36 +13,34 @@ aliases:
   - /docs/pulumi-cloud/esc/faq
 ---
 
-## Is it safe to use Pulumi ESC for my production stacks?
+## Why did Pulumi launch ESC?
 
-Yes, Pulumi ESC is served using the same [highly secure](/security/pulumi-cloud-security-whitepaper.pdf) and highly available infrastructure as the rest of Pulumi Cloud. The service is currently in preview as we are rapidly iterating and improving the feature set. It is unlikely that we will make any breaking changes. Should we introduce a new API, we are dedicated to ensuring optimal compatibility to prevent any interruptions to your infrastructure.
+We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC brings the same software engineering approach to secrets and configuration, allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks. just like Pulumi IaC. It can be used for all applications and infrastructure - with or without Pulumi IaC.
```

Comment:
> We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks.

Add a comma before causing.

> We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets, causing sprawl and duplications across stacks.

-----

> Pulumi ESC brings the same software engineering approach to secrets and configuration, allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks. just like Pulumi IaC.

This is garbled.

Reply:
Updated